.github/workflows/ci.yml

```diff
@@ -363,10 +361,8 @@ jobs:
     - name: Update pip
       run: python3 -m pip install --upgrade pip
 
-    - name: Setup CMake 3.18
-      uses: jwlawson/actions-setup-cmake@v1.5
-      with:
-        cmake-version: 3.18
```

Comment:
😳 Whoops, bit late, after I just merged, but this is 3.18.

Was there a reason for it to be 3.18? I guess not, because you also removed the name change, @henryiii?

Reply:
If it's passing, I think it's okay, since we test each version elsewhere. But I didn't see that it was 3.18, I thought it was an inconsistently named 3.19.

Reply:
There might have been a bug in an early version of 3.19, but hopefully if so, it is fixed.

Reply:
Tests are passing, so we should be fine?